sechub-cli/src/daimler.com/sechub/cli/job.go

```diff
@@ -103,46 +103,43 @@ func getSecHubJobState(context *Context, checkOnlyOnce bool, checkTrafficLight b
 		} else {
 			// PROGRESS bar ... 50 chars with dot, then next line...
 			if newLine {
-				fmt.Print("                             ")
+				sechubUtil.PrintIfNotSilent("                             ", context.config.quiet)
```

Comment:
Instead of using white spaces you can use the `Repeat` function: https://golang.org/pkg/strings/#Repeat

Reply:
Good tip. Changed to strings.Repeat().